virtool/references/api.py

```diff
@@ -188,7 +188,7 @@ async def update(req):
         "user_id": user_id,
     }
 
-    task = await req.app["tasks"].add(UpdateRemoteReferenceTask, context=context)
+    task = await get_data_from_req(req).tasks.add(UpdateRemoteReferenceTask, context=context)
```

Comment:
Let's take the opportunity to change this from `add` to `create` to match the rest of the data layer.